src/lib/CollisionPrevention/CollisionPrevention.cpp

```diff
@@ -58,171 +72,212 @@ CollisionPrevention::~CollisionPrevention()
 	}
 }
 
-void CollisionPrevention::_updateOffboardObstacleDistance(obstacle_distance_s &obstacle)
+hrt_abstime CollisionPrevention::getTime()
```

Comment:
Why is this needed?

Reply:
So that we can fake the time in the tests to check that timeouts work, and that old data is dropped.